ddl/cluster.go

```diff
@@ -84,6 +85,9 @@ func ValidateFlashbackTS(ctx context.Context, sctx sessionctx.Context, flashBack
 	if oracle.GetTimeFromTS(flashBackTS).After(oracle.GetTimeFromTS(currentTS)) {
 		return errors.Errorf("cannot set flashback timestamp to future time")
 	}
+	if oracle.GetTimeFromTS(flashBackTS).After(expression.GetMinSafeTime(sctx)) {
+		return errors.Errorf("cannot set flashback timestamp to greater than resolved ts from store")
```

Comment:
The term `Resolved TS` could be confusing to the user, maybe we can rephrase it to:

```suggestion
		return errors.Errorf("cannot set flashback timestamp to too close to present")
```